src/tools/Move.ts

```diff
@@ -0,0 +1,77 @@
+import { Tool, Tracker } from '@/tools/Tool'
+import Konva from 'konva'
+import { CanvasElement } from '@/types/Canvas'
+import throttle from 'lodash.throttle'
+
+export default class FreeDraw implements Tool {
+  // eslint-disable-next-line
+  private group: Konva.Group
+  constructor (public readonly name: string,
+               public temporary: boolean) {
```

Comment:
is temporary applicable for this tool? Do we need to have a property if it won't be used?

Reply:
I made it a standard for all tools to include the name and the temporary variable in the constructor